src/components/course-list/CourseList.jsx

```diff
@@ -1,37 +1,34 @@
 import React, { Component } from 'react';
 import { CourseItem } from '../course-item';
-import { getAll } from "../../db/db";
 import { withRouter } from "react-router-dom";
+import listCoursesWrapper from '../../hoc/ListCoursesWrapper';
 
 
 class CourseListBlock extends Component {
```

Comment:
That logic would be better move to separate method with the name like `renderCourses()`.
And write it below the `render()` method. Because we should always write custom logic like methods or other below the render method and write logic like `componentDidMount()` or other logic of the React.Component ot static methods in above.
```
class Some extends React.Component {
    static propTypes ...
    static defaultProps ...
    
    constructor() ...
   
    componentDidMount() ....
   
    render() ....
    
    renderCourses() ....
}   
````

Reply:
It not only about that component, it like common code style

Reply:
fixed